rollup.config.mjs

```diff
@@ -45,23 +47,23 @@ if (process.env.BUILD === 'dist') {
     },
     external: (id) => !id.startsWith('.') && !path.isAbsolute(id),
     plugins: [
-      nodeResolve({browser: true}),
       babel({
-        babelHelpers: 'bundled',
+        babelHelpers: 'external',
+        skipPreflightCheck: true,
```

Comment:
were preflight checks causing an issue? https://github.com/rollup/plugins/tree/master/packages/babel#skippreflightcheck
They seem harmless, if not beneficial.

Reply:
Which issue? No issue. Preflight check just does not work with external helpers which are usually incorrect to use.

Reply:
And remind me why the helpers need to be external :thinking:

Reply:
https://github.com/micromark/micromark/pull/36#discussion_r527732630

Reply:
Might be an XY problem: I wanted to ensure we don’t get helpers in there. Maybe there’s a better way to ensure that!

Reply:
Well, it may not always fail. But I can add plugin which will fail if babel/runtime import occurs.

Reply:
Or just watch carefully for plugin list. Helpers are appeared only from builtin plugins usually.